migrations/3_ServiceRequest.js

```diff
@@ -12,5 +16,10 @@ module.exports = function(deployer, network, accounts) {
     let _minStake = 100000000;
     let _maxStakers = 100;
 
-    deployer.deploy(Token, {overwrite: false}).then((TokenInstance) => deployer.deploy(ServiceRequest, TokenInstance.address, _minStake, _maxStakers));
+    // AGI-I Contract deployment -- Will be deleted once AGI-2 is deployed - Kept it for compatibility only
+    deployer.deploy(Token, { overwrite: false, gas: 4000000 }).then((TokenInstance) => deployer.deploy(ServiceRequest, TokenInstance.address, _minStake, _maxStakers));
+
```

Comment:
We should be uncommenting this code right?

Reply:
Yes we need to uncomment the code once we have deployed the npm package. Otherwise due to the current package referring to the package will fail.
Right now we are referring to the current version of the token contract and once we have the new token contract package will update the migration/deployment code.